plugins/catalog/src/js/repositories/data/repositoriesModel.js

```diff
@@ -4,6 +4,13 @@ import "rxjs/add/operator/map";
 import RepositoryList from "#SRC/js/structs/RepositoryList";
 import { makeExecutableSchema } from "graphql-tools/dist/index";
 
+// Streams we get our data from
+import {
+  liveFetchRepositories,
+  addRepository,
+  deleteRepository
+} from "#PLUGINS/catalog/src/js/repositories/data/PackageRepositoryClient";
```

Comment:
VSCode does that for me sometimes. So I wonder if it's your IDE or you've deliberately chosen to use the `#PLUGINS` alias?

Reply:
No, I did not. I think for imports inside a plugin, it makes no sense to use the plugins path. I think it was a global switch and replace or the editor. 

I appreciate you did not block because of this one. I will move further like this, this time because I know there will be opportunities to correct that in the future as we move forward, and because it is not a huge deal.